### PR TITLE
Update Flow parser to v0.21

### PR DESCRIPTION
**Pull request by jeffmo**
Includes various new syntax and early-stage feature options

### CODE REVIEW

File: src/parsers/js/esprima.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default {
   },
 
   parse(esprima, code) {
-    return esprima.parse(code, options);
+    return espima.parse(code, options);
```
Comment (hzoo) [inline src/parsers/js/esprima.js]: typo?

Comment (jeffmo) [inline src/parsers/js/esprima.js]: I knew that red bull was a bad idea...

File: src/parsers/js/flow.js [hunk 2]
```diff
@@ -55,10 +44,9 @@ export default {
...
     });
   },
-	*/
 };
 
-function changeOption(name, {target: {value}}) {
-  options[name] = value;
-  LocalStorage.setParserSettings(ID, options);
+function changeOption(name, {target}) {
+	options[name] = target.checked;
```
Comment (jeffmo) [inline src/parsers/js/flow.js]: Ugh, just upgraded vim too -- will fix this weird tab stuff

**Comment by jeffmo** [comment]
(Ok, I think it's ready now)

**Comment by fkling** [comment]
Site is updated!

**Pull request merged**